<!--
    Launcher for basic stereo camera which publishes images
-->
<launch>
  <!-- Machine settings. See http://www.ros.org/wiki/roslaunch/XML/machine -->
  <!--machine name="local_alt" address="localhost" default="true" /-->
  <!-- stereocamera -->
  <node pkg="minorucam" type="minorucam_disp" name="minorucam_disp" respawn="true" output="screen">
    <!-- left camera device -->
    <param name="dev_left" value="/dev/video1" />
    <!-- right camera device -->
    <param name="dev_right" value="/dev/video0" />
    <!-- image width -->
    <param name="width" value="320" />
    <!-- image height -->
    <param name="height" value="240" />
    <!-- Frames per second -->
    <param name="fps" value="30" />
    <!-- Flip the left image -->
    <param name="flip_left" value="false" />
    <!-- Flip the right image -->
    <param name="flip_right" value="false" />
    <!-- filenames -->
    <param name="calibration_filename" value="/home/pi/calibration.txt" />
    <param name="left_camera_filename" value="left_camera.txt" />
    <param name="right_camera_filename" value="right_camera.txt" />
    <param name="stereo_camera_filename" value="stereo_camera.txt" />
  </node>
</launch>

